app/controllers/users_controller.rb

```diff
@@ -1,9 +1,6 @@
 class UsersController < ApplicationController
-  def show
-
-  end
 
-  def new
+  def show
     @user = User.find_by(params[:id])
```

Comment:
thêm code xử lý nếu `@user` bị nil nè 
`return if @user`
`flash[:warning] = "User not found"`
`redirect_to root_path`